src/index.html

```diff
@@ -1,13 +1,61 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
         content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+    <link href="https://fonts.googleapis.com/css2?family=Comforter+Brush&family=Roboto:wght@300;400;500&display=swap" rel="stylesheet">
```

Comment:
```suggestion
<link rel="preconnect" href="https://fonts.googleapis.com"> 
<link 
  rel="preconnect" 
  href="https://fonts.gstatic.com" 
  crossorigin
> 
<link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
```

Look carefully, you hadn't copied font for font-weight-700, which is used for price (69 999 uah)